src/browser/context-menu-template.js

```diff
@@ -0,0 +1,83 @@
+var reload = function(item, focusedWindow) {
+  if (focusedWindow) {
+    focusedWindow.webContents.reload();
+  }
+};
+
+var comment = function(item, focusedWindow) {
+  var selection = document.getSelection();
+  if (selection.toString().length > 0) {
+    var textbox = document.getElementById("post_textbox");
+    textbox.value = "> " + selection;
+    textbox.focus();
+  }
+};
+
+var changeTemplateForOSx = function(obj){
+  //OSx uses the 'selector' key whereas Windows/Linux uses 'role'
+  for (var i=0; i < obj.length; i++) {
+    if (obj[i].role) {
+      obj[i].selector = obj[i].role;
+      delete obj[i].role;
+    }
+  }
+  return obj;
+};
+
+var template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    role: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    role: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    role: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    role: 'copy:'
+  },
+  {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    role: 'paste:'
+  },
+  {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    role: 'selectAll:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Reload',
+    accelerator: 'CmdOrCtrl+R',
+    click: reload
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Comment',
+    accelerator: 'Shift+CmdOrCtrl+C',
+    click: comment
```

Comment:
I pulled it at 11pm. :-) I'll figure it out. Probably just a user error.
On Jan 29, 2016 7:51 AM, "3-john-4" <notifications@github.com> wrote:

> In src/browser/context-menu-template.js
> <https://github.com/HackerHappyHour/matterfront/pull/82#discussion_r51261992>
> :
>
> > +  },
> > +  {
> > +    type: 'separator'
> > +  },
> > +  {
> > +    label: 'Reload',
> > +    accelerator: 'CmdOrCtrl+R',
> > +    click: reload
> > +  },
> > +  {
> > +    type: 'separator'
> > +  },
> > +  {
> > +    label: 'Comment',
> > +    accelerator: 'Shift+CmdOrCtrl+C',
> > +    click: comment
>
> Obvious question but did you pull the latest? I just added this commit
> late yesterday.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HackerHappyHour/matterfront/pull/82/files#r51261992>.
>